transparency/transparency.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) The go-boot authors. All Rights Reserved.
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+package transparency
+
+import (
+	"fmt"
+	"io/fs"
+
+	"github.com/usbarmory/go-boot/uefi/x64"
+	"github.com/usbarmory/boot-transparency/engine/sigsum"
+	"github.com/usbarmory/boot-transparency/policy"
+	"github.com/usbarmory/boot-transparency/transparency"
+)
+
+// Boot transparency configuration.
+var Config struct {
+	Status string
+
+	bootPolicy    []byte
+	witnessPolicy []byte
+	proofBundle   []byte
+	submitKey     []byte
+	logKey        []byte
+}
+
+const (
```

Comment:
These constants should live outside this package and be used as arguments to the package methods, or probably much better as fields of *Config given their number, which per earlier comment can be passed to provide arguments and keep state.

This allows users of go-boot transparency package to use its functions with paths others than the one fixed here.